windows-agent/internal/proservices/proservices_test.go

```diff
@@ -133,6 +138,10 @@ func TestRegisterGRPCServices(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			t.Parallel()
 
+			if tc.wantServices == nil {
+				tc.wantServices = defaultServices
```

Comment:
I am a little bit less fan to have default "want" when empty/nil contrary to input data. WDYT about being explicit for `wantServices` in "Success with WSL net adapter" case? (especially as here, there is only one instance)

Reply:
I'm just fine with the proposal.